### PR TITLE
[StyleCop] Fix all warnings in DateTime files (spacing and commas)

### PR DESCRIPTION
**Pull request by ceciliaavila**
- Added trailing commas and parenthesis when needed.
- Fixed spacing.
- Reordered methods and using properties.

### CODE REVIEW

File: .NET/Microsoft.Recognizers.Text.DateTime/Utilities/TimeZoneUtility.cs [hunk 1]
```diff
@@ -44,9 +44,7 @@ public static bool ShouldResolveTimeZone(ExtractResult er, DateTimeOptions optio
 
             if (er.Data is Dictionary<string, object>)
             {
-                var metadata = er.Data as Dictionary<string, object>;
-
-                if (metadata != null && metadata.ContainsKey(Constants.SYS_DATETIME_TIMEZONE))
+                if (er.Data is Dictionary<string, object> metadata && metadata.ContainsKey(Constants.SYS_DATETIME_TIMEZONE))
```
Comment (Bill7zz) [inline .NET/Microsoft.Recognizers.Text.DateTime/Utilities/TimeZoneUtility.cs]: I suggest keeping the metadata variable to make the if statement more descriptive.

File: .NET/Microsoft.Recognizers.Text.DateTime/Utilities/RangeTimexComponents.cs [hunk 2]
```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Globalization;
+using System.Text;
+using DateObject = System.DateTime;
+
+namespace Microsoft.Recognizers.Text.DateTime
+{
+    public class RangeTimexComponents
```
Comment (VictorGrycuk) [inline .NET/Microsoft.Recognizers.Text.DateTime/Utilities/RangeTimexComponents.cs:10]: On the fields of this class, VS is reporting warning [SA1401](https://github.com/DotNetAnalyzers/StyleCopAnalyzers/blob/master/documentation/SA1401.md) "Field should be private". I think they missing `{ get; set; }`.

**Review by VictorGrycuk — CHANGES_REQUESTED** [review]
Please validate the warning in my comment, otherwise let me know and I'll approve the PR.

**Review by Bill7zz — APPROVED** [review]
LGTM

File: .NET/Microsoft.Recognizers.Text.DateTime/Utilities/RangeTimexComponents.cs (hunk 2 above)
Comment (ceciliaavila) [reply .NET/Microsoft.Recognizers.Text.DateTime/Utilities/RangeTimexComponents.cs:10]: Feedback applied.

**Review by JuanAr — APPROVED** [review]
👍

**Pull request closed without merging**